roles/bootstrap/tasks/directories.yml

```diff
@@ -0,0 +1,12 @@
+---
+
+- name: Create required system directories
+  file:
+    path: "{{ item }}"
+    state: directory
+  become: true
+  with_items:
+    - /data
+    - /srv
+    - /var/log
+    - /usr/src/packages
```

Comment:
same as above, however not much of debian person, so it could be this doesn't actually exist on a fresh OS install